regression_tests/regression_test.go

```diff
@@ -0,0 +1,131 @@
+//go:build regression
+// +build regression
+
+package regressiontests
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/regression_tests/services"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRegression_TransactionsWorkflow(t *testing.T) {
+	const adminXPriv = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+	const actor1Alias = "Actor1RegressionTest"
+	const actor2Alias = "Actor2RegressionTest"
+
+	ctx := context.Background()
+	leaderService1, actorService1 := initTestServices(ctx, t, &services.LeaderServiceConfig{
+		EnvURL:     lookupEnvOrDefault("CLIENT_ONE_URL", ""),
```

Comment:
What do you think about providing the defaults so they don't need to be set/exported anywhere when running locally?

Reply:
Good point Damian :) This should be discussed with the rest of the team about what values to provide.